usecases/auth/authorization/conv/casbin_types.go

```diff
@@ -47,12 +47,13 @@ var (
 		authorization.Root:   CRUD,
 	}
 	actions = map[string]string{
-		CRUD:                 "manage",
-		CRU:                  "manage",
-		authorization.CREATE: "create",
-		authorization.READ:   "read",
-		authorization.UPDATE: "update",
-		authorization.DELETE: "delete",
+		CRUD:                           "manage",
+		CRU:                            "manage",
+		authorization.ROLE_SCOPE_MATCH: "manage",
```

Comment:
let's just call it "role_scope_match"

Reply:
not sure what you mean , this what will be printed as permission `manage_roles`